src/python/pants/pantsd/pailgun_server.py

```diff
@@ -56,8 +56,14 @@ class PailgunHandler(PailgunHandlerBase):
 
   def _run_pants(self, sock, arguments, environment):
     """Execute a given run with a pants runner."""
+    # For the pants run, we want to log to stderr.
+    # TODO Might be worth to make contextmanagers for this?
```

Comment:
Yea, that would probably be good. But at the very least, I'd wrap this in a `try: finally:` before landing.

Reply:
Stale comment.